era_5g_relay_network_application/server.py

```diff
@@ -413,7 +419,7 @@ def main(args=None) -> None:
     topics_incoming_list = load_entities_list("TOPICS_FROM_CLIENT")  # Topics that are received from the client
     services_incoming_list = load_entities_list("SERVICES_FROM_CLIENT")  # Services that are called from the client
     actions_to_provide = load_entities_list("ACTIONS_FROM_CLIENT")  # Actions provided by this server to the client
-    transforms_to_listen = load_transform_list("TRANSFORMS_FROM_CLIENT")  # TFs that are received from the client
+    transforms_to_listen = load_transform_list("TRANSFORMS_TO_CLIENT")  # TFs that are received from the client
```

Comment:
So this was a bug? From variable names and the comment, it sounds reasonable to me like it was :-)

Reply:
Yes, it was a bug; the comment must also be updated. The variable name is correct because it is a list of the transforms to which the server should "listen to" locally and send them to the client.

Reply:
Are you going to update the comment?

Reply:
I did, but I did not save the file :) now it has been updated